src/v2/view-builder/views/TerminalView.js

```diff
@@ -49,6 +53,9 @@ const Footer = BaseFooter.extend({
     if (this.options.appState.containsMessageWithI18nKey(RETURN_LINK_EXPIRED_KEY)) {
       return getBackToSignInLink(this.options.settings);
     }
+    if (this.options.appState.containsMessageStartingWithI18nKey(SAFE_MODE_KEY)) {
```

Comment:
Can we add a derived variable in appstate to check this? 

Like 

'isSafeMode' 
and check if the remediation is just skip over there ?

Reply:
We only know if it is in safe mode based on the error message key. There may be a skip remediation, or it may be empty

Reply:
@gowthamidommety-okta Is there any harm in showing the link always rather thank checking the i18n key.

Reply:
Multiple flows use this view. my only concern is that the skip link will show up there. This check is what email is doing above too..